deploy.sh

```diff
@@ -1,17 +1,16 @@
 #!/bin/bash
+set -euo pipefail
 
+export IP_WHITELIST=${POISE_WHITELIST}
 export KUBE_NAMESPACE=${ENVIRONMENT}
 export KUBE_SERVER=${KUBE_SERVER}
-
-if [[ -z ${VERSION} ]] ; then
-    export VERSION=${IMAGE_VERSION}
-fi
-if [[ -z ${DOMAIN} ]] ; then
-    export DOMAIN="cs"
+export KUBE_TOKEN=${KUBE_TOKEN}
+export VERSION=${VERSION}
+  
+export DOMAIN="cs"
+if [ $(echo ${KUBE_NAMESPACE} | sed 's/-.*//') == "wcs" ]; then
```

Comment:
All those Drone string manipulation functions are lifted from Bash, which means we can do:
```suggestion
if [ ${KUBE_NAMESPACE%-*} == "wcs" ]; then
```

Reply:
Done 👍🏼